src/bindings/event.ts

```diff
@@ -1,4 +1,4 @@
-import { Context } from "probot";
+import { Context as ProbotContext } from "probot";
```

Comment:
You sure about the typecasting here? Seems like you're killing the point of the typing?